scripts/release/package_audit.rb

```diff
@@ -0,0 +1,233 @@
+version_tag = ARGV[0] || "latest"
+
+# Manual list - MAKE SURE UP TO DATE especially base images
+containers = [
+  { name: "ballaerospace/cosmosc2-ruby:#{version_tag}", base_image: "alpine:3.15.0", apk: true, gems: true },
```

Comment:
We have the ALPINE_VERSION in the .env file. Can you use that here?

Reply:
Future improvement would be to pull all this info from the Dockerfiles and the .env file